src/ScheduleFunctions.cpp

```diff
@@ -818,23 +818,112 @@ string schedule_to_source(Function f,
     if (compute_at.is_inline()) {
         ss << ".compute_inline()";
     } else {
+        string store_var = store_at.var;
```

Comment:
I'd prefer to call this "store_loc" or "store_name" as "store_var" implies to me the thing is a var.